src/foundation.cs

```diff
@@ -1776,6 +1784,11 @@ interface NSDateIntervalFormatter {
 
 		[Export ("stringFromDate:toDate:")]
 		string StringFromDate (NSDate fromDate, NSDate toDate);
+
+		[Watch (3,0)][TV (10,0)][Mac (10,12)][iOS (10,0)]
+		[Export ("stringFromDateInterval:")]
+		[return: NullAllowed]
+		string ToString (NSDateInterval dateInterval);
```

Comment:
@spouliot Shouldn't we, at the same time, `XAMCORE_4_0`-ify the other methods: `stringForObjectValue:`, `stringFromDateComponents:`, `stringFromTimeInterval:` so they use `ToString` too (which is a much better binding).